examples/apps/ai-collab/src/app/page.tsx

```diff
@@ -51,7 +49,7 @@ export async function createAndInitializeContainer(): Promise<
 export default function TasksListPage(): JSX.Element {
 	const [selectedTaskGroup, setSelectedTaskGroup] = useState<SharedTreeTaskGroup>();
 	const [treeView, setTreeView] = useState<TreeView<typeof SharedTreeAppState>>();
-	const [userPresenceGroup, setUserPresenceGroup] = useState<UserPresence>();
+	const [presence, setPresence] = useState<IPresence>();
```

Comment:
Using `IPresence` "at the top" so we can leverage the events for updated data and for participants disconnecting, as well as the current participant's sessionId.